api/napi/index.ts

```diff
@@ -147,13 +147,35 @@ export class ArrayModel<T> implements Model<T> {
     }
 }
 
+/**
+ * This interface describes the common public API of a Slint component.
```

Comment:
(Please double check the formatting before applying the suggestion, it might not compile cleanly with tsdoc)

```suggestion
 * This interface describes the public API of a Slint component that is common to all instances. Use this to
 * show() the window on the screen, access the window and subsequent window properties, or start the
 * Slint event loop with run().
```